pyfar/classes/filter.py

```diff
@@ -164,6 +164,15 @@ def init_state(self, state='zeros'):
         self._state = state
```

Comment:
should we remove the state='zero' here, because in this method we set the state as the array and not as a config string.
this funtion init_state is more a set_state anyway, even its not checking the inputs, (which should be done acutally). maybe shift the check of state from the init into this one.

overall its very confusing, that the state in the init is an array and in init_state (except for Filter) is a string. Maybe we should consider another name, but thats an other story.

Reply:
moved into #757 
but maybe we should remove the wron default here state='zero'

Reply:
Yes, this is wrong. Fortunately it is called from the child classes in any case